BUILD

```
@@ -23,3 +24,13 @@ cc_test(
     ":board",
   ],
 )
+
+cc_test(
+  name = "boardtest",
+  size = "small",
+  srcs = ["boardtest.cpp"],
+  deps = [
+    "@com_google_googletest//:gtest_main",
+    ":board",
+  ],
+)
```

Let's add a newline to the end of this file.
```suggestion
)

```